toolchain/build-toolchain.sh

```diff
@@ -11,21 +11,21 @@ log() {
 }
 
 # targets for libphoenix and phoenix-rtos-kernel for installing headers
-declare -A TOOLCHAN_TO_PHOENIX_TARGET=(
-    [arm-phoenix]="armv7a7-imx6ull"
+declare -A TOOLCHAN_TO_PHOENIX_TARGETS=(
+    [arm-phoenix]="armv7a7-imx6ull armv7m7-imxrt106x armv7m4-stm32l4x6"
```

Comment:
Hmm, I think that Zynq uses other FPU than imx6ull - isn't that crucial for libphoenix build?

Also different mcpu passed. I'm not sure how it matter in terms of toolchain multilib.

Reply:
the FPU doesn't matter unless its of a different breed (eg. simd/no-simd). I assumed it's the same arch, but it seems it's slightly different:

![image](https://github.com/phoenix-rtos/phoenix-rtos-build/assets/3939983/a95c2056-4436-494a-8ab5-5d67399adf87)

I'll add explicit zynq libphoenix building

Reply:
heeh, so it's matter of newer cortex-a7 core, not a FPU type… Wonder if it can change code generation...

Anyways, worth to note that libc will be build with optimisation on specific cortex cpu. Probably it's good enough but maybe worth noticing.

Reply:
previously we compiled it with specific optimizations and installed in toolchain on first target build within `phoenix-rtos-project` so the used "optimized" version of libphoenix was the same. 

This change enables future not-building-libphoenix approach.

Reply:
also `v7ve` is `7-a` + virtualization extensions, so unless we would start using virtualization it shouldn't (hopefully) make any difference ;)